.gitignore

```diff
@@ -30,3 +30,7 @@
 *.vrb
 *.xdy
 *.tdo
+*.fls
+demo/test.pdf
+demo/test.pdfpc
+auto/
```

Comment:
what is `auto/` ?

Reply:
I'm not sure; for some reason latex compilers seem to generate this directory for me sometimes.